store/tikv/region_cache.go

```diff
@@ -70,6 +71,7 @@ type RegionStore struct {
 	workStoreIdx int32    // point to current work peer in meta.Peers and work store in stores(same idx)
 	stores       []*Store // stores in this region
 	storeFails   []uint32 // snapshots of store's fail, need reload when `storeFails[curr] != stores[cur].fail`
+	followers    []int32  // followers' index in this region
```

Comment:
We can just use a leader index instead.
When we trying to find a follower, we can just skip the leader.

Reply:
It will have to check current store index every single time that way. Wouldn't it be even more cumbersome?

Reply:
The `followers` don't have more information, we can remove it to save memory.
And the computation cost to avoid `workStoreIdx` is very little.

Reply:
We are using seed passed in here. Which one are we going to use when it's pointing to leader? Whichever follower it chooses as a fallback, how to make sure it can really balance the load? In addition, arrays for a million regions only cost dozens of MB. Compare to other places, the footprint of this array is actually negligible.

Reply:
```go
numPeers := 5
followerIdx := seed % (numPeers-1)
if followerIdx >= workStoreIdx {
    followerIdx++
    if followerIdx == numPeers {
        followerIdx = 0
    }
}
```


if the index >= workStoreIdx

Reply:
Brilliant, let's do it this way.

Reply:
Actually, `followerIdx == numPeers` will always be false.

This will do

```go
numPeers := 5
followerIdx := seed % (numPeers-1)
if followerIdx >= workStoreIdx {
    followerIdx++
}
```

Reply:
Done